Testscripts/Linux/SetupRDMA.sh

```diff
@@ -382,32 +382,35 @@ function Main() {
 	cd ~
 	LogMsg "Proceeding Intel MPI Benchmark test installation"
 
-	# install Intel MPI benchmark package
-	LogMsg "Cloning mpi-benchmarks repo, $intel_mpi_benchmark"
-	git clone $intel_mpi_benchmark
-	Verify_Result
-	LogMsg "Cloned Intel MPI Benchmark gitHub repo"
-	cd mpi-benchmarks/src_c
-	LogMsg "Building Intel MPI Benchmarks tests"
-	make -j $(nproc)
-	Verify_Result
+	# Intel MPI has its own IMB-MPI1, IMB-NBC and IMB-RMA binaries
+	if [ $mpi_type != "intel" ]; then
+		# install Intel MPI benchmark package
+		LogMsg "Cloning mpi-benchmarks repo, $intel_mpi_benchmark"
+		git clone $intel_mpi_benchmark
+		Verify_Result
+		LogMsg "Cloned Intel MPI Benchmark gitHub repo"
+		cd mpi-benchmarks/src_c
+		LogMsg "Building Intel MPI Benchmarks tests"
+		make -j $(nproc)
```

Comment:
just more as an FYI as I also learned recently, we can drop the nproc use to -j.
From make man page:
"If the -j option is given without an argument, make will not limit the number of jobs that can run simultaneously."
and this was seen to use all cores just by running "make -j"